deploy/docker/polus-railyard/Dockerfile

```
@@ -0,0 +1,3 @@
+FROM node:12
```

Pretty easy way to lower docker file size.

```suggestion
FROM node:12-alpine
```
Not sure what this is used for but if you can.  node:12-alpine is about 25 MB while node:12 is about 900 MB.

---

Good point. I did not think much when I chose the base image. This is used in Jenkins CI to fill in the template Dockerfile from the custom yaml definition